packages/util/src/constants.ts

```diff
@@ -1 +1,13 @@
+import { codeBlock } from './code-block';
+
 export const VALUE_IS_UNDEFINED = 'VALUE_IS_UNDEFINED';
+
+export function getCopyrightHeader(): string {
```

Comment:
We have a ticket in the backlog to have a generator-util at some point in time. Until then we have duplicated all code an added a todo. I would suggest to do the same here, because otherwise we will have to make breaking changes to move this to the new package.